src/form-components/RangeEditor.js

```diff
@@ -9,64 +10,125 @@ import PropTypes from 'prop-types';
+                        >&gt;</button>
+                        <button
+                            className="btn btn-primary ml-2"
+                            id={this.props.id}
+                            data-id={this.props.dataId}
+                            type="button"
+                            onClick={this.props.handler}
+                            value={this.btnStep(1, Number(this.props.step) * 10 || 0.1)}
+                        >&gt;&gt;&gt;</button>
+                    </div>
+                </div>
+                <div className="input-group">
+                    {this.props.showOverrideButton && (
+                        <div className="form-check form-check-inline">
+                            <label className="form-check-label">
+                                <input
+                                    data-id={this.props.dataId}
+                                    data-override={this.props.overrideValue}
+                                    className="form-check-input override"
+                                    type="radio"
+                                    onChange={this.props.handler}
+                                    checked={this.props.value === this.props.overrideValue} />
+                                {this.props.overrideLabel}
+                            </label>
+                        </div>
+                    )}
+                    {this.props.showOverride2Button && (
+                        <div className="form-check form-check-inline">
+                            <label className="form-check-label">
+                                <input
+                                    data-id={this.props.dataId}
+                                    data-override={this.props.override2Value}
+                                    className="form-check-input override"
+                                    type="radio"
+                                    onChange={this.props.handler}
+                                    checked={this.props.value === this.props.override2Value} />
+                                {this.props.override2Label}
+                            </label>
+                        </div>
+                    )}
                 </div>
-            )}
-        {this.props.showOverride2Button && (
-            <div className="form-check form-check-inline">
-                <label className="form-check-label">
-                    <input
-                        data-id={this.props.dataId}
-                        data-override={this.props.override2Value}
-                        className="form-check-input override"
-                        type="radio"
-                        onChange={this.props.handler}
-                        checked={this.props.value === this.props.override2Value} />
-                    {this.props.override2Label}
-                </label>
             </div>
-        )}
-        </div>
         </React.Fragment>;
     }
+    btnStep(sign, strength) {
```

Comment:
Intricate logic like this is a perfect candidate for unit testing. I would move this function out into `utils.js`, parameterizing `min`, `max`, and `value`, and write some tests for it in `utils.test.js`.